bin/dfx-canister-nm

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -euo pipefail
+SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+PATH="$SOURCE_DIR:$PATH"
+. "$SOURCE_DIR/versions.bash"
+
+print_help() {
+  cat <<-EOF
+
+	Print functions exported by a canister.
+	EOF
+}
+
+# Source the clap.bash file ---------------------------------------------------
+source "$SOURCE_DIR/clap.bash"
+# Define options
+# Source the output file ----------------------------------------------------------
+source "$(clap.build)"
+
+# The only meaningful export from canister wasms is function names.
+canister_nm=(wasm-nm -je)
+
+canister_wasm="$1"
```

Comment:
Does clap provide help on positional args or should it be added manually?

Reply:
It doesn't.  I'd like to add support for positional args, as it would give us automatic documentation.  At one time I was hesitant to add support as array handling, limited as it is, is not supported in decade old macbash.  But if we have a policy of getting people to install an up to date bash, that reason disappears.

Reply:
In that case I think it's worth adding a usage line to `print_help` so it's clear how the parameter is passed.